### PR TITLE
Configurable keyword case + pretty printer

### PR DESCRIPTION
**Pull request by FreHu**
@sbcgua you should like this
- `keywords_upper` renamed to `keyword_case` which has a `style: upper | lower` property
- pretty printer will uppercase or lowercase depending on the style

### CODE REVIEW

**Comment by larshp** [comment]
did you try using Enums? https://www.typescriptlang.org/docs/handbook/enums.html

comparing with strings, `style === "lower"` makes it difficult to refactor and finding references. But I'm not quite sure how/if enums works with the json schema

**Comment by FreHu** [comment]
The editor/type system is smart enough to provide intellisense for the values, but I didn't think of the weaker refactoring support. If you give them string values, the schema will stay the same.

**Comment by larshp** [comment]
cool, looks better to me 👍

File: src/rules/keyword_case.ts [hunk 1]
```diff
@@ -59,15 +66,15 @@ export class KeywordsUpper extends ABAPRule {
 
     for (const statement of file.getStatements()) {
       if (statement.get() instanceof Unknown
-          || statement.get() instanceof MacroContent
-          || statement.get() instanceof MacroCall
-          || statement.get() instanceof Comment) {
+        || statement.get() instanceof MacroContent
```
Comment (larshp) [inline src/rules/keyword_case.ts]: I actually like having the extra spaces here, so its easy for the eye to find the contents, will probably add a tslint/eslint rule sometime in the future

Comment (FreHu) [reply src/rules/keyword_case.ts]: It's an inconsistency between our formatters, do you happen to know where I can configure it to behave the same?

Comment (larshp) [reply src/rules/keyword_case.ts]: I format these things manually 😄 but not a big problem for now

**Pull request merged**